docs/setup/self-hosted/docker-desktop.mdx

```diff
@@ -1,26 +1,38 @@
 ---
-title: Setup as a Docker Extension
-sidebarTitle: Docker Extension
+title: Docker Desktop Extension for MindsDB
+sidebarTitle: Docker Desktop
 ---
 
-This is work in progress but [Ajeet Singh Raina](https://github.com/ajeetraina) has built a great docker extension, should look like this
+MindsDB provides an extension for Docker Desktop that facilitates running MindsDB on Docker Desktop.
 
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/34368930/230012152-c60bc8a8-3c00-48c9-961c-6d10421e7e26.png"/>
-</p>
+<Tip>
+Visit the [GitHub repository for MinsdDB Docker Desktop Extension](https://github.com/mindsdb/mindsdb-docker-extension) to learn more.
+</Tip>
+
+## Prerequisites
+
+Before proceeding, ensure you have installed Docker Desktop, following the [official Docker Desktop documentation](https://www.docker.com/products/docker-desktop/).
 
+## Setup
 
+This setup of MindsDB uses the `mindsdb/mindsdb:latest` Docker image, which is a lightweight Docker image of MindsDB that comes with [these integrations preloaded](https://github.com/mindsdb/mindsdb/blob/staging/default_handlers.txt).
 
-<Info>
-## Install
-Follow the instructions here if you want to try it:
+Follow the steps to set up MindsDB in Docker Desktop.
 
-<CardGroup cols={1}>
+## Install MindsDB
 
-    <Card title="MindsDB Docker extension guide" icon="link" href="https://github.com/collabnix/mindsdb-docker-extension"></Card>
+Go to the Extensions page in Docker Desktop and search for MindsDB.
 
+Install the MindsDB extension.
+
+Access MindsDB inside Docker Desktop.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/34368930/230012152-c60bc8a8-3c00-48c9-961c-6d10421e7e26.png"/>
+</p>
 
-</CardGroup>
+## Install dependencies
 
-</Info>
+In the MindsDB editor, go to *Settings* and *Manage Integrations*.
 
+Select integrations you want to use and click on *Install*.
```

Comment:
Should we maybe add a picture of the page here (and maybe down in the Docker doc) too? Just wondering, not mandatory.